cdap-app-fabric/src/main/java/io/cdap/cdap/internal/profile/ProfileService.java

```diff
@@ -214,7 +238,8 @@ public void deleteProfile(ProfileId profileId)
    *
    * @param namespaceId the id of the namespace
    */
-  public void deleteAllProfiles(NamespaceId namespaceId)
+  @AuthEnforce(entities = "profileId", enforceOn = ProfileId.class, actions = Action.EXECUTE)
```

Comment:
This was supposed to be removed since we have granular check at profileId. Also mismatch between @Name and entities field.